frontend/src/pages/index.tsx

```diff
@@ -1,13 +1,5 @@
-import { ReactElement } from "react";
+import Programs from "./programs";
 
-import Login from "./login";
-
-import Landing from "@/components/Landing";
-
-export default function MyApp() {
-  return <Login />;
+export default function Home() {
```

Comment:
Can we modify this so that the base url still routes to login? I don't want to mess up the current flow if we were to merge this in.